### ISSUE TITLE
ALVR Meta Quest 1 support on headset

### ISSUE THREAD

**Issue by pbek** [alvr-org/ALVR#2532]
## Description

I followed the instructions in the `README.md` for my Meta Quest 1 which sent me to https://www.meta.com/en-gb/experiences/alvr/7674846229245715/, but the app only supports Meta Quest 2+ and I wasn't able to install it on my Meta Quest 1. The App Page on my device said `Not available`.

So is the Meta Quest 1 really supported?

## General Troubleshooting
- [x] I carefully followed the instructions in the [README](https://github.com/alvr-org/ALVR/blob/master/README.md) and successfully completed the setup wizard
- [x] I read the [ALVR GitHub Wiki](https://github.com/alvr-org/ALVR/wiki)

### Hardware

Meta Quest 1

**Comment by The-personified-devil** [comment]
Yes, but you need the apk from the github release. Perhaps there should be a distinction in rhe Readme.

**Comment by pbek** [comment]
Ah, thank you. I didn't even know you can sideload APKs on the quest.
